kafka-connect-neo4j/doc/docker-compose.yml

```diff
@@ -1,86 +1,83 @@
+      NEO4J_server_memory_heap_max__size: "4G"
 
   zookeeper:
-    image: confluentinc/cp-zookeeper
+    image: confluentinc/cp-zookeeper:7.3.0
     hostname: zookeeper
     container_name: zookeeper
     ports:
-    - "2181:2181"
+      - "2181:2181"
     environment:
       ZOOKEEPER_CLIENT_PORT: 2181
       ZOOKEEPER_TICK_TIME: 2000
 
   broker:
-    image: confluentinc/cp-enterprise-kafka
+    image: confluentinc/cp-server:7.3.0
     hostname: broker
     container_name: broker
     depends_on:
-    - zookeeper
+      - zookeeper
     ports:
-    - "9092:9092"
-    expose:
-    - "9093"
+      - "9092:9092"
+      - "9101:9101"
     environment:
-      KAFKA_ADVERTISED_LISTENERS: PLAINTEXT://broker:9093,OUTSIDE://localhost:9092
-      KAFKA_LISTENER_SECURITY_PROTOCOL_MAP: PLAINTEXT:PLAINTEXT,OUTSIDE:PLAINTEXT
-      KAFKA_LISTENERS: PLAINTEXT://0.0.0.0:9093,OUTSIDE://0.0.0.0:9092
-      CONFLUENT_METRICS_REPORTER_BOOTSTRAP_SERVERS: broker:9093
-
-      # workaround if we change to a custom name the schema_registry fails to start
-      KAFKA_INTER_BROKER_LISTENER_NAME: PLAINTEXT
-
       KAFKA_BROKER_ID: 1
       KAFKA_ZOOKEEPER_CONNECT: 'zookeeper:2181'
+      KAFKA_LISTENER_SECURITY_PROTOCOL_MAP: PLAINTEXT:PLAINTEXT,PLAINTEXT_HOST:PLAINTEXT
+      KAFKA_ADVERTISED_LISTENERS: PLAINTEXT://broker:29092,PLAINTEXT_HOST://localhost:9092
       KAFKA_METRIC_REPORTERS: io.confluent.metrics.reporter.ConfluentMetricsReporter
       KAFKA_OFFSETS_TOPIC_REPLICATION_FACTOR: 1
       KAFKA_GROUP_INITIAL_REBALANCE_DELAY_MS: 0
-      CONFLUENT_METRICS_REPORTER_ZOOKEEPER_CONNECT: zookeeper:2181
+      KAFKA_CONFLUENT_LICENSE_TOPIC_REPLICATION_FACTOR: 1
+      KAFKA_CONFLUENT_BALANCER_TOPIC_REPLICATION_FACTOR: 1
+      KAFKA_TRANSACTION_STATE_LOG_MIN_ISR: 1
+      KAFKA_TRANSACTION_STATE_LOG_REPLICATION_FACTOR: 1
+      KAFKA_JMX_PORT: 9101
+      KAFKA_JMX_HOSTNAME: localhost
+      KAFKA_CONFLUENT_SCHEMA_REGISTRY_URL: http://schema-registry:8081
+      CONFLUENT_METRICS_REPORTER_BOOTSTRAP_SERVERS: broker:29092
       CONFLUENT_METRICS_REPORTER_TOPIC_REPLICAS: 1
       CONFLUENT_METRICS_ENABLE: 'true'
       CONFLUENT_SUPPORT_CUSTOMER_ID: 'anonymous'
 
-  schema_registry:
-    image: confluentinc/cp-schema-registry
-    hostname: schema_registry
-    container_name: schema_registry
+  schema-registry:
+    image: confluentinc/cp-schema-registry:7.3.0
+    hostname: schema-registry
+    container_name: schema-registry
     depends_on:
-    - zookeeper
-    - broker
+      - broker
     ports:
-    - "8081:8081"
+      - "8081:8081"
     environment:
-      SCHEMA_REGISTRY_HOST_NAME: schema_registry
-      SCHEMA_REGISTRY_KAFKASTORE_CONNECTION_URL: 'zookeeper:2181'
+      SCHEMA_REGISTRY_HOST_NAME: schema-registry
+      SCHEMA_REGISTRY_KAFKASTORE_BOOTSTRAP_SERVERS: 'broker:29092'
+      SCHEMA_REGISTRY_LISTENERS: http://0.0.0.0:8081
 
   connect:
-    image: confluentinc/cp-kafka-connect
+    image: cnfldemos/cp-server-connect-datagen:0.6.0-7.3.0
```

Comment:
is this the right image?

Reply:
yeah, doesn't look so but I also don't know why this is the image in the documents

Reply:
I think we can keep it as `confluentinc/cp-kafka-connect:7.3.0`.

Reply:
There is an error in connect instance logs like 
```
2024-07-19 09:12:39 [2024-07-19 07:12:39,449] ERROR Uncaught exception in REST call to /v1/metadata/id (org.apache.kafka.connect.runtime.rest.errors.ConnectExceptionMapper)
2024-07-19 09:12:39 javax.ws.rs.NotFoundException: HTTP 404 Not Found
```
when I tried `confluentinc/cp-kafka-connect:7.3.0` 

but `confluentinc/cp-server-connect:7.3.0` seems to work seamlessly